control-plane/pkg/reconciler/trigger/controller.go

```diff
@@ -119,6 +126,9 @@ func NewController(ctx context.Context, watcher configmap.Watcher, configs *conf
 		},
 	})
 
+	reconciler.SecretTracker = impl.Tracker
+	secretinformer.Get(ctx).Informer().AddEventHandler(controller.HandleAll(reconciler.SecretTracker.OnChanged))
```

Comment:
Did we simply forget this OR we just need this tracking because of the changes here?

Reply:
because of the changes here